metadata.tex

```diff
@@ -182,6 +185,9 @@
       }
       \end{varwidth}
     };
+    \begin{pgfonlayer}{background}
+      \draw [shade, blur shadow={shadow blur steps=5, shadow opacity=15}] (current bounding box.north west) rectangle (current bounding box.south east);
```

Comment:
Can you experiment with this parameter? In the rendered images, there seems to be a 1px border around the shadows.
```suggestion
      \draw [shade, blur shadow={shadow blur steps=8, shadow opacity=15}] (current bounding box.north west) rectangle (current bounding box.south east);
```